packages/framework-types/src/concepts/read-model.ts

```diff
@@ -6,6 +6,7 @@ export interface ReadModelInterface {
   id: UUID
   boosterMetadata?: {
     version: number
+    optimisticConcurrencyValue?: string | number
```

Comment:
Would it make sense to always use strings and cast them to numbers on the providers that require them?

Reply:
I'd leave it like this to avoid the casting tbh